perf/benchmark/run_benchmark_job.sh

```diff
@@ -117,7 +117,7 @@ function setup_fortio_and_prometheus() {
 
 #TODO: add stackdriver filter
 function prerun_v2_nullvm() {
-  export SET_OVERLAY="--set values.telemetry.enabled=true --set values.telemetry.v1.enabled=false --set values.telemetry.v2.enabled=true --set values.telemetry.v2.prometheus.enabled=true"
+  export SET_OVERLAY="--set components.telemetry.enabled=false --set values.telemetry.enabled=true --set values.telemetry.v1.enabled=false --set values.telemetry.v2.enabled=true --set values.telemetry.v2.prometheus.enabled=true"
```

Comment:
I'm confused, why preprun_v2 telemetry.enabled=false, why not true?

Reply:
components.telemetry.enabled is controlling mixer v1 deployment/configs etc. This is confusing right now, plz refer to this issue for more context: https://github.com/istio/istio/issues/21048